CMakeLists.txt

```diff
@@ -6,7 +6,7 @@
 #   schmieder.matthias@gmail.com
 
 cmake_minimum_required(VERSION 3.13)
-project(minuzip-ng
+project(minizip-ng
```

Comment:
This ignores `MZ_PROEJCT_SUFFIX`..

Reply:
Only now I found that projest() is actually declared below my changes. Usually it is declared on top of the cmake files.
A bit odd way of using that statement.
I'll be busy in next day or two so feel free to close that PR and implement necessary chanhes basing on my PR.

Reply:
I don't use any linux OS as my primary operating system, so my knowledge of install directories is rather limited.